RFS/src/main/java/org/opensearch/migrations/bulkload/workcoordination/OpenSearchWorkCoordinator.java

```diff
@@ -397,14 +397,11 @@ public void completeWorkItem(
         }
     }
 
-    private int numWorkItemsArePending(
-        int maxItemsToCheckFor,
+    private int numWorkItemsNotYetCompleteInternal(
         Supplier<IWorkCoordinationContexts.IPendingWorkItemsContext> contextSupplier
     ) throws IOException, InterruptedException {
         try (var context = contextSupplier.get()) {
             refresh(context::getRefreshContext);
-            // TODO: Switch this to use _count
-            log.warn("Switch this to use _count");
```

Comment:
For future reference: The author of this P found that _count was slower than the search